pool/store/sqlite_store.py

```diff
@@ -116,23 +116,56 @@ def _row_to_farmer_record(row) -> FarmerRecord:
         )
 
     async def add_farmer_record(self, farmer_record: FarmerRecord, metadata: RequestMetadata):
```

Comment:
I think we need to modify pg_store.py as well as the logic has been updated in sqlite_store.py

Reply:
I have updated pg_store in the last commit with applying "upsert" for adding a new farmer.